extension/src/experiments/columns/model.ts

```diff
@@ -197,12 +172,18 @@ export class ColumnsModel extends PathSelectionModel<Column> {
 
     this.data = columns
 
-    if (this.columnOrderState.length === 0) {
```

Comment:
[B] If `this.columnOrderState.length > 0` but `columnOrderState` does not contain the current params and metrics then `getSummaryColumnOrder` gives unexpected results.

Reply:
This is actually still the case because there can be stale data in the status object. I will make an update.